pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+## Why did you make this change?
+(cite Issue # OR provide rationalization of changes if no issue exists)
```

Comment:
If there are relevant errors or traceback, I think they should belong in the issue that a PR is addressing. Otherwise, I think our PRs may become excessively verbose.

Reply:
There's no obligation to create an issue in order to open a bugfix PR (and I don't think there should be).